### PR TITLE
Schedule the merge conflict labelling to run daily

### PR DESCRIPTION
**Pull request by ASpoonPlaysGames**
Runs at 4:20 UTC every day

It was taking a while for labels to get removed, sometimes

### CODE REVIEW

**Comment by ASpoonPlaysGames** [comment]
If you want to test this, dont commit anything to main until 4:20 tomorrow, when https://github.com/R2Northstar/NorthstarLauncher/pull/617 should lose the label

File: .github/workflows/merge-conflict-auto-label.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - main
+  schedule:
+    - cron: "20 4 * * *"
```
Comment (GeckoEidechse) [inline .github/workflows/merge-conflict-auto-label.yml:7]: Setting this to run pretty soon for testing ^^

```suggestion
    - cron: "40 20 * * *"
```

**Review by GeckoEidechse — APPROVED** [review]
Not really testable until merged.

**Comment by GeckoEidechse** [comment]
Merged and should run within the next minute ^^

**Pull request merged**